content/en/about/supported-releases/index.md

```diff
@@ -5,19 +5,57 @@ weight: 35
 icon: cadence
 ---
 
-This page lists the status and timeline for currently supported releases. Supported releases of Istio include releases that are in the active
-maintenance window and are patched for security and bug fixes. Subsequent patch releases on a LTS release do not contain backward incompatible
-changes. For more information refer to the [Istio support policy](../release-cadence/).
+This page lists the status, timeline and policy for currently supported releases. Supported releases of Istio include releases that are in the active
+maintenance window and are patched for security and bug fixes. Subsequent patch releases on a minor release do not contain backward incompatible
+changes.
+
+* [Support Policy](#support-policy)
+* [Naming scheme](#naming-scheme)
+* [Support status of Istio releases](#support-status-of-istio-releases)
+* [Releases without known Common Vulnerabilities and Exposures (CVEs)](#releases-without-known-Common-Vulnerabilities-and Exposures)
+
+## Support policy
+
+We produce new builds of Istio for each commit. Around once a quarter, we build a minor release and
+run through several additional tests as well as release qualification. We issue patch releases for issues found in minor
+releases.
+
+The different types represent a different product quality level and different level of support
+from the Istio team. In this context, *support* means that we will produce patch releases for critical issues and
```

Comment:
```suggestion
from the Istio community. In this context, *support* means that the community will patch releases for critical issues and
```

Reply:
Updated in https://github.com/istio/istio.io/pull/8991/files